src/execution/__tests__/schema-test.js

```diff
@@ -91,7 +91,7 @@ describe('Execute: Handles execution with a complex schema', () => {
     function article(id) {
       return {
         id,
-        isPublished: 'true',
```

Comment:
one more example why current coercion rules looks strange and confusing:
`"true" => true`
`"false" => false`